src/test/java/com/stripe/model/SubscriptionTest.java

```diff
@@ -1,157 +1,25 @@
-
-		HashMap<String, Object> params = new HashMap<String, Object>();
-		List<HashMap<String, Object>> items = new ArrayList<HashMap<String, Object>>();
-
-		HashMap<String, Object> itemA = new HashMap<String, Object>();
-		itemA.put("plan", "gold");
-		itemA.put("quantity", 1);
-
-		HashMap<String, Object> itemB = new HashMap<String, Object>();
-		itemB.put("plan", "silver");
-		itemB.put("quantity", 2);
-
-		items.add(itemA);
-		items.add(itemB);
-
-		params.put("items", items);
-
-		subscription.update(params);
-
-		verifyPost(Subscription.class, "https://api.stripe.com/v1/subscriptions/test_sub", params);
-		verifyNoMoreInteractions(networkMock);
-	}
-
-	@Test
-	public void testCreate() throws StripeException {
-		HashMap<String, Object> params = new HashMap<String, Object>();
-		params.put("customer", "test_cus");
-		params.put("plan", "gold");
-
-		Subscription subscription = Subscription.create(params);
-
-		verifyPost(Subscription.class, "https://api.stripe.com/v1/subscriptions", params);
-		verifyNoMoreInteractions(networkMock);
-	}
-
-	@Test
-	public void testCreateWithItems() throws StripeException {
-		Map<String, Object> params = new HashMap<String, Object>();
-		List<HashMap<String, Object>> items = new ArrayList<HashMap<String, Object>>();
-
-		HashMap<String, Object> itemA = new HashMap<String, Object>();
-		itemA.put("plan", "gold");
-		itemA.put("quantity", 1);
-
-		HashMap<String, Object> itemB = new HashMap<String, Object>();
-		itemB.put("plan", "silver");
-		itemB.put("quantity", 2);
-
-		items.add(itemA);
-		items.add(itemB);
-
-		params.put("customer", "test_cus");
-		params.put("items", items);
-
-		Subscription subscription = Subscription.create(params);
-
-		verifyPost(Subscription.class, "https://api.stripe.com/v1/subscriptions", params);
-		verifyNoMoreInteractions(networkMock);
-	}
-
-	@Test
-	public void testCancel() throws StripeException {
-		Subscription subscription = new Subscription();
-		subscription.setId("test_sub");
-
-		subscription.cancel(null);
-
-		verifyDelete(Subscription.class, "https://api.stripe.com/v1/subscriptions/test_sub");
-		verifyNoMoreInteractions(networkMock);
-	}
-
-	@Test
-	public void testSetDeleteDiscount() throws StripeException {
-		Subscription subscription = new Subscription();
-		subscription.setId("test_sub");
-
-		Discount discount = new Discount();
-		discount.setId("test_dis");
-		subscription.setDiscount(discount);
-		Assert.assertEquals("test_dis", subscription.getDiscount().id);
-
-		subscription.deleteDiscount();
-
-		verifyDelete(Discount.class, "https://api.stripe.com/v1/subscriptions/test_sub/discount");
-		verifyNoMoreInteractions(networkMock);
+	public void testDeserializeWithExpansions() throws Exception {
+		String[] expansions = { "*" };
+		String data = getFixture("/v1/subscriptions/sub_123", expansions);
+		Subscription object = APIResource.GSON.fromJson(data, Subscription.class);
+		assertNotNull(object);
+		assertNotNull(object.getCustomer());
```

Comment:
This is what most model tests will look like: one "basic" test that just tries to deserialize a fixture and one "expansions" test that uses the special `*` to tell the stub that we want every expandable field expanded. We then try to deserialize the response to make sure that all our `ExpandableField` instances run. I do a basic `assertNotNull` on a single expandable field as well just as a basic check that it did something.